integtest/spec/all_books_change_detection_spec.rb

```diff
@@ -545,6 +571,53 @@ def chapter(index)
           include_examples 'toc and version drop down'
           let(:current) { 'master' }
         end
+        context 'because we add a branch to the book and specify ' \
+                '--keep_hash and --sub_dir' do
```

Comment:
Neither of these reproduced the problem either but they were useful it making sure I didn't break anything while working on it.